spec/app/domain/authentication/authn-azure/application_identity_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Authentication::AuthnAzure::ApplicationIdentity do
+  include_context "azure setup"
+
+  let(:subscription_id_annotation) { double("SubscriptionIdAnnotation") }
```

Comment:
this is defined also in `validate_azure_annotations_spec.rb`. Should it move to "azure setup"?